provision-osx/tasks/install_xcode.yml

```diff
@@ -55,14 +62,40 @@
 # Check here for versions already installed
 -
   name: Install xcode versions
-  shell: "source ~/.bash_profile && xcversion install {{ item }}"
+  shell: "{{ xcversion_bin.stdout }} install {{ item }}"
   args:
     executable: /bin/bash
   with_items: "{{ xcode_versions }}"
   when: xcode_install_user is defined and xcode_install_password is defined
   register: xcode_install
   failed_when: xcode_cli_tools_install.rc != 0 and xcode_cli_tools_install.stdout.find('already installed') == -1
   changed_when: xcode_cli_tools_install.stdout.find('already installed') == -1
+  become: yes
   environment:
     XCODE_INSTALL_USER: "{{ xcode_install_user }}"
     XCODE_INSTALL_PASSWORD: "{{ xcode_install_password }}"
+
+-
+  name: Check xcode output
+  debug: msg="Checking {{ item.cmd }} output"
+  with_items: "{{ xcode_install.results }}"
+  failed_when: item.stderr.find('The specified Apple developer account credentials are incorrect.') != -1
+
+-
+  name: Check xcode symlink
+  stat:
+    path: /Applications/Xcode.app
+  register: xcode_symlink
+
+-
+  name: Set default xcode symlink path
+  set_fact:
+    xcode_symlink_path: "/Applications/Xcode-{{ xcode_default_version }}.app"
+
+-
+  name: Set default xcode symlink
```

Comment:
I noticed that the `xcversion` command will set a symlink to the last installed xcode  version which I don't think is the ideal behavior.